GitUI/Editor/FileViewer.cs

```diff
@@ -726,6 +741,25 @@ void DetectDefaultEncoding()
 
         // Private methods
 
+        private static bool ViewIsDiff(ViewMode viewMode)
```

Comment:
The name is very odd, how about this?
```suggestion
        private static bool IsDiffView(ViewMode viewMode)
```